src/LodeRunner.Core/Events/LoadResultEventArgs.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace LodeRunner.Core.Events
+{
+    /// <summary>
+    /// Represents the LoadResultEventArgs and contains the main functionality of the class.
+    /// </summary>
+    /// <seealso cref="System.EventArgs" />
+    public class LoadResultEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LoadResultEventArgs"/> class.
+        /// </summary>
+        /// <param name="startTime">Start time of the TestRun.</param>
+        /// <param name="completedTime">Time TestRun completed.</param>
+        /// <param name="testRunId">Id of the TestRun.</param>
+        /// <param name="total">Number of total requests.</param>
+        /// <param name="failed">Number of failed requests.</param>
+        /// <param name="errorMessage">Error message (if exception occurred).</param>
+        public LoadResultEventArgs(DateTime startTime, DateTime completedTime, string testRunId, int total, int failed, string errorMessage = "")
+        {
+            this.StartTime = startTime;
```

Comment:
Looks like the intention is to only set Public properties in the Constructor, if so then change the setter accessibility level to private to avoid accidental update.

Reply:
I omitted the setters so they can only be updated in the constructor for now.  It's possible with future refactors that we will need the setters (or some of them) to be public, but I'm good leaving them out for now.